### PR TITLE
Hyperdiffusion on rho and rho-theta

### PR DESCRIPTION
**Pull request by jiahe23**

### CODE REVIEW

**Review by szy21 — APPROVED** [review]
Looks good, thanks! Should we add it to ci?

**Comment by jiahe23** [comment]
bors try

File: examples/3dsphere/baroclinic_wave_utilities.jl [hunk 1]
```diff
@@ -259,6 +259,81 @@ function baroclinic_wave_ρθ_remaining_tendency!(dY, Y, p, t; κ₄)
...
+function held_suarez_ρθ_hyperdiffusion_ρ_ρθ_remaining_tendency!(dY, Y, p, t; κ₄)
+    @unpack cχθ, cχuₕ, cuvw, cω³, fω¹², fu¹², fu³, cf, cK, cp, cΦ = p
+
+    cρ = Y.Yc.ρ # density on centers
+    cρθ = Y.Yc.ρθ # potential temperature on centers
+    cuₕ = Y.uₕ # Covariant12Vector on centers
+    fw = Y.w # Covariant3Vector on faces
+
+    # Update w at the bottom
+    # fw = -g^31 cuₕ/ g^33 ????????
```
Comment (valeriabarra) [inline examples/3dsphere/baroclinic_wave_utilities.jl:271]: I know this has been copied over multiple times now and it was not inserted in this PR, but I have been curious to know what does this comment mean?

Comment (jiahe23) [reply examples/3dsphere/baroclinic_wave_utilities.jl:271]: @simonbyrne  might know

**Review by akshaysridhar — APPROVED** [review]

**Pull request merged**